src/Symfony/Component/Ldap/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.0.0
```

Comment:
IIRC this can go into 4.3 minor

Reply:
This makes sense; addressed in latest revision.